hack/golint.sh

```diff
@@ -5,10 +5,18 @@ set -eux
 IS_CONTAINER=${IS_CONTAINER:-false}
 CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
 
+GOLANGCI_LINT=bin/golangci-lint
+if [ ! -f "${GOLANGCI_LINT}" ]; then
+    curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.39.0
```

Comment:
Line 26 refers to a specific golint image that we maintain on quay. Could we put this command in the Dockerfile that builds that image?

Reply:
@dhellmann How about using official golangci-lint docker image as a stage in builder image?

Reply:
Sure, that makes sense, too.